ground/gcs/src/plugins/uavsettingsimportexport/importsummary.cpp

```diff
@@ -2,6 +2,7 @@
  ******************************************************************************
  *
  * @file       importsummary.cpp
+ * @author     dRonin, http://dRonin.org/, Copyright (C) 2015
```

Comment:
I didn't make any changes. git cherry-pick flubbed because the original commit history was composed of changes in commits and changes in merges. The end result was having to merge all changes into one commit. Dunno why even though I amended the commit to put Joao as the author it wound up ultimately reverting back to me.